bitwarden_license/src/Scim/Controllers/v2/GroupsController.cs

```diff
@@ -135,20 +137,8 @@ public async Task<IActionResult> Post(Guid organizationId, [FromBody] ScimGroupR
     [HttpPut("{id}")]
     public async Task<IActionResult> Put(Guid organizationId, Guid id, [FromBody] ScimGroupRequestModel model)
     {
-        var group = await _groupRepository.GetByIdAsync(id);
-        if (group == null || group.OrganizationId != organizationId)
-        {
-            return new NotFoundObjectResult(new ScimErrorResponseModel
-            {
-                Status = 404,
-                Detail = "Group not found."
-            });
-        }
-
-        group.Name = model.DisplayName;
-        await _groupService.SaveAsync(group);
-        await UpdateGroupMembersAsync(group, model, false);
-        return new ObjectResult(new ScimGroupResponseModel(group));
+        var scimGroupResponseModel = await _putGroupCommand.PutGroupAsync(organizationId, id, model);
```

Comment:
Let's return the `group` and construct the `ScimGroupResponseModel` in the controller instead.